docs/tutorial.part-1.rst

```diff
@@ -108,3 +108,46 @@ installed when you installed populus.
     tests/test_greeter.py::test_named_greeting PASSED
 
 You should see something akin to the output above with three passing tests.
+
+Behind the scenes, populus uses a pytest plugin that creates a populus project object, and then provide this object,
+(and it's derived objects), to the test functions via a pytest fixture.
+
+Thus, tests run for a specific project.
+
+If you run py.test from within the project directory, populus will assume that the current working directory
+is the project you want to test, and the fixtures will be based on this directory.
+
+The same is true if you provide pytest one positional argument for testing, which is the project directory:
+
+.. code-block:: bash
+
+    $ py.test /path/to/my/project/
+
+Here, populus will provide the fixtures based on the project at ``/path/to/my/project/``. Pytest will also find the tests in that directory.
+Note that populus and py.test look for their files separatly. Pytest looks for and collects the tests, populus for the the populus.json and other project files.
+
+
+When you want to run tests that are saved outside the project directory, you will have to explictly provide the project directory.
```

Comment:
Since this is a tutorial, it feels like too much information to go into this.  Maybe a small mention and then a link to the `docs/testing.rst` docs on the subject?